sync.go

```diff
@@ -80,44 +80,50 @@ func RunSync(configFile string) error {
 	}
 
 	maxNameLength := config.MaxSyncSetNameLength()
-	var errors []string
+	var alertList []string
 
 	// Iterate through SyncSets and process changes
 	for i, syncSet := range config.SyncSets {
 		if syncSet.Name == "" {
 			msg := "configuration contains a set with no name"
-			errors = append(errors, msg)
+			alertList = append(alertList, msg)
 		}
 		prefix := fmt.Sprintf("[ %-*s ] ", maxNameLength, syncSet.Name)
 		syncSetLogger := log.New(os.Stdout, prefix, 0)
 		syncSetLogger.Printf("(%v/%v) Beginning sync set", i+1, len(config.SyncSets))
 
 		// Apply SyncSet configs (excluding source/destination as appropriate)
-		err = source.ForSet(syncSet.Source)
-		if err != nil {
-			msg := fmt.Sprintf(`Error setting source set on syncSet "%s": %s`, syncSet.Name, err)
-			syncSetLogger.Println(msg)
-			errors = append(errors, msg)
+		if err = source.ForSet(syncSet.Source); err != nil {
+			err = fmt.Errorf(`Error setting source set on syncSet "%s": %w`, syncSet.Name, err)
+			syncSetLogger.Println(err)
```

Comment:
I'm thinking this should be moved to inside `handleSyncError`, but then you'd either need to pass `syncSetLogger` or change it to `log.Println()`.  Personally, I don't see any issues with using `log.Println` as the output and Flags for log are set at the beginning of `RunSync()`, or is there a reason we're using `log.New()`?

Reply:
The `syncSetLogger` is customized, so if it were to use simply `log.Println` it wouldn't use the custom prefix. Or maybe you're suggesting that for each pass through the loop, the global logger gets redefined. I suppose that would work, but I haven't tried it.

Reply:
Hmm, what about setting the prefix each loop then reverting at end?  That said current implementation is good too.